WORKING_GROUPS.md

```diff
@@ -27,6 +27,10 @@ Its responsibilities are:
 * Develop and maintain a build and automation system for `iojs.org`.
 * Ensure the site is regularly updated with changes made to `io.js` like
 releases and features.
+* Documentation: maintain the docs build process and publish to `iojs.org`. See https://github.com/iojs/doc-tool (the source API docs are maintained and managed in core, not through this WG.)
+* General marketing and social media initiatives (proposed sub-WG)
+* Fostering io.js evangelism efforts (empowering speakers with materials, maintaining a list of interested speakers/interviewees, education, etc.)
```

Comment:
sub-WG isn't really a thing. this is covered in WORKING_GROUPS but basically once something has a big enough community behind it we spin out a WG and when that work was already handled by a WG then that WG *and* the TC have to approve the charter, but the WG concept is essentially flat (although we expect a lot of cross-collaboration between groups as they find necessary).